student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ] Since the Student class inherits from the Person we should provide the needed parameters for creating a Person and also pass them up to the parent class:

  ```
  def initialize(age, classroom, name = "Unknown", parent_permission: true)  
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```